botocore/docs/method.py

```diff
@@ -217,7 +217,7 @@ def document_model_driven_method(
         method_intro_section.writeln('')
 
     # Add the example section.
-    example_section = section.add_new_section('example')
+    example_section = section.add_new_section('request-example')
```

Comment:
Just wondering why did this and response one below it need to change?

Reply:
There was no easy way to determine which section was which in order to determine the section title. They were both called `example`. With the updated names i can map `request-example` to `Request Syntax`.